style.css

```diff
@@ -593,3 +597,652 @@ form {
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+    margin-top: 15px;
+  }
+
+  .Tonic4 {
+    display: block;
+    height: 46px;
+    width: 496px;
+    left: 0;
+    top: 0;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #091e42;
+  }
+
+  .offer4 {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    flex-direction: row;
+    gap: 12px;
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  .offer4 .canopy {
+    height: 24px;
+    width: 43px;
+    left: 0;
+    top: 8px;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+    text-transform: capitalize;
+  }
+
+  .offer4 .backend {
+    height: 24px;
+    width: 140px;
+    left: 0;
+    top: 8px;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #6b778c;
+    text-transform: capitalize;
+  }
+
+  .offer4 .year {
+    height: 24px;
+    width: 38px;
+    left: 0;
+    top: 8px;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .offer4 .point {
+    height: 8px;
+    width: 8px;
+    left: 55px;
+    top: 16px;
+    border-radius: 0;
+    margin-top: 9px;
+  }
+
+  .skills2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    gap: 8px;
+    padding-left: 0;
+    height: 44px;
+  }
+
+  .htl {
+    text-transform: lowercase;
+  }
+
+  .offer2 {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    flex-direction: row;
+    gap: 25px;
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+    margin-top: 10px;
+  }
+
+  .description2 {
+    display: block;
+    height: 70px;
+    width: 430px;
+    left: 24px;
+    top: 146px;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+    margin-top: 15px;
+  }
+
+  .description4 {
+    height: 53px;
+    width: 496px;
+    left: 24px;
+    top: 146px;
+    display: block;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+    margin-top: 15px;
+  }
+
+  .container1 .seeproject {
+    border: 1px solid #dfe1e6;
+  }
+
+  .container2 .seeproject {
+    border: 1px solid #dfe1e6;
+  }
+
+  .container4 .seeproject {
+    border: 1px solid #dfe1e6;
+  }
+
+  .container3 .seeproject {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-decoration: none;
+    text-align: center;
+    letter-spacing: 0.03em;
+    padding: 12px;
+    color: #fff;
+    background: #396df2;
```

Comment:
- [ ] Please consider updating the button style as shown in the screenshots below, the default style for this button should be the same as the other buttons, in the Figma design is shown as blue background just for clarification for the interaction styling when the mouse hovers over the project card 👍🏼 .

### Your Design:
![image](https://user-images.githubusercontent.com/68030297/228996990-26944799-d06c-4579-97fb-d14039f076f8.png)

### How it suppose to look like:
![image](https://user-images.githubusercontent.com/68030297/228997062-491b6513-95d8-4bcf-ab54-276dd40d02b3.png)